src/NLog.Web/README.md

```diff
@@ -5,11 +5,18 @@
 
 Supported platforms:
 
+The .NET 3.5 build supports the following .NET releases
```

Comment:
Instead of doing this splitting, then it should just state that `- .NET Framework 3.5 - 4.8` are supported.

Reply:
Changed